src/huggingface_hub/_upload_large_folder.py

```diff
@@ -93,6 +95,17 @@ def upload_large_folder_internal(
     repo_url = api.create_repo(repo_id=repo_id, repo_type=repo_type, private=private, exist_ok=True)
     logger.info(f"Repo created: {repo_url}")
     repo_id = repo_url.repo_id
+    # 2.1 Check if xet is enabled to set batch file upload size
+    is_xet_enabled = (
+        is_xet_available()
+        and api.repo_info(
+            repo_id=repo_id,
+            repo_type=repo_type,
+            revision=revision,
+            expand="xetEnabled",
+        ).xet_enabled
+    )
+    target_upload_batch_size = TARGET_NB_FILES_XET_UPLOAD_BATCH_SIZE if is_xet_enabled else 1
```

Comment:
```suggestion
    upload_batch_size = UPLOAD_BATCH_SIZE_XET if is_xet_enabled else UPLOAD_BATCH_SIZE_LFS
```

Reply:
In the PR in general the batch size is named:
- nb_files_upload_batch_size
- upload_batch_size
- target_upload_batch_size
- TARGET_NB_FILES_XET_UPLOAD_BATCH_SIZE

depending on where it's defined. I believe always naming it `upload_batch_size` is less confusing and still understandable (a "batch size" is very often a "target batch size" in practice). And define `UPLOAD_BATCH_SIZE_XET` / `UPLOAD_BATCH_SIZE_LFS` for constant values.

Could you update the naming in the whole PR? :pray: